src/lifecycles/load.js

```diff
@@ -47,6 +47,10 @@ export function toLoadPromise(app) {
           return app;
         }
 
+        if (appOpts.devtools && appOpts.devtools.overlays) {
+          app.devtools.overlays = {...app.devtools.overlays, ...appOpts.devtools.overlays}
+        }
```

Comment:
consuming overlay options from the application by pulling them from the exported devtools option if it exists

Reply:
See https://github.com/CanopyTax/single-spa/pull/273#discussion_r254878956